tests/test_subsample.py

```diff
@@ -0,0 +1,97 @@
+"""Tests subsampling
+
+These tests verifies:
+    - Binning correctly
+    - Subsampling correctly
+
+PREREQUISITES
+Spin up Python 3.6 virtualenv, install Python dependencies in requirements.txt
+and Firestore emulator must be running, see PR26 for instructions
+(https://github.com/broadinstitute/scp-ingest-pipeline/pull/26)
```

Comment:
As in `test_ingest_files`, I suspect Firestore might not be required to run the tests in this file.

Reply:
6344fd2